tests/test_functional.py

```diff
@@ -0,0 +1,399 @@
+# Copyright (c) 2014 Google, Inc.
+# Copyright (c) 2014-2016 Claudiu Popa <pcmanticore@gmail.com>
+
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/COPYING
+
+"""Functional full-module tests for PyLint."""
```

Comment:
Code in this file can't possibly pass our linters, can you make sure that this file is linted and locally-disable the rules taht it violates for now?